### DISCUSSION TITLE
Change browser tab label for Discussions

### DISCUSSION THREAD

**Discussion by mwichmann** [community/community#130499] [Discussions]
### Select Topic Area

Product Feedback

### Body

Very simple-minded request: could the tab text for Discussions change to match those for Issues and Pull Requests?  For example, for a project I work on, we see:
```
[Pull Requests - SCons/scons - GitHub]
[Issues - SCons/scons - GitHub]
[SCons/scons - Discussions - GitHub]
```
Since when you have lots of tabs open they get squished by the browser, it would be helpful for finding if the word "Discussions" came first like the other two.

**Comment by LiteBrite82** [comment]
Hi @mwichmann 

We appreciate your feedback and your valuable input. We want to be transparent about our current priorities. You can gain insights into the projects we’re currently focusing on for the upcoming year by checking our [public roadmap](https://github.com/orgs/github/projects/4247). While smaller fixes may not always be explicitly listed, they often get addressed when related work is in progress.

Regarding your specific request, it’s important to note that it’s not actively being worked on at the moment. However, this doesn’t mean we won’t consider it in the future; it’s simply not on our development roadmap right now.

We’d like to emphasize that as a valued GitHub user, your feedback in this Discussion space, along with appropriate tags and examples, plays a crucial role in shaping the development of GitHub. While we can’t guarantee immediate action on every piece of feedback, please rest assured that we are attentive to your suggestions and continuously strive to enhance the GitHub experience.
While we do review all incoming feedback unfortunately due to the volume of feedback, we may not be able to respond to each one individually at this time. However, we strongly encourage you to continue providing your valuable input as it greatly helps us prioritize and improve GitHub.

Your contributions are instrumental in making GitHub better for all users. Thank you for taking the time to share your thoughts with us, even if the particular request is not currently in active development. Your input is truly valuable and contributes to the continuous improvement of GitHub.
